package.json

```diff
@@ -109,6 +109,7 @@
     "js-levenshtein": "^1.1.6",
     "json-2-csv": "^5.4.0",
     "jsonwebtoken": "^9.0.0",
+    "langchain": "^0.2.5",
```

Comment:
Could you please add `OPENAI_API_KEY` to .env.example? Also in twenty-website's documentation (we will depreciate twenty-docs in a couple days and are moving to twenty-website's docs)